src/App.tsx

```diff
@@ -1,5 +1,135 @@
-const App = () => {
-  return <div>App</div>;
-};
+import {
+  Chart as ChartJS,
+  LinearScale,
+  CategoryScale,
+  BarElement,
+  PointElement,
+  LineElement,
+  Legend,
+  Tooltip,
+  Filler,
+  LineController,
+  BarController,
+} from 'chart.js';
+import { Chart } from 'react-chartjs-2';
+
+import { useValues } from './hooks/useValues';
+
+ChartJS.register(
+  LinearScale,
+  CategoryScale,
+  BarElement,
+  PointElement,
+  LineElement,
+  Legend,
+  Tooltip,
+  Filler,
+  LineController,
+  BarController,
+);
+
+export function App() {
+  const chartData = useValues();
+  const chartObj = chartData?.response;
+  const barArrData = [];
+  const areaArrData = [];
+
+  Object.entries(chartObj || {}).map(([key, value]) =>
+    barArrData.push({ x: key, y: value.value_bar, z: value.id }),
+  );
+
+  Object.entries(chartObj || {}).map(([key, value]) =>
+    areaArrData.push({ x: key, y: value.value_area, z: value.id }),
+  );
```

Comment:
```suggestion
  chartObj && Object.entries(chartObj).map(([key, value]) =>{
    barArrData.push({ x: key, y: value.value_bar, z: value.id });
    areaArrData.push({ x: key, y: value.value_area, z: value.id });
  };
```

이렇게 리펙토링 해보는 방법이 어떨까요? 🤔

- 동일한 사이클을 2번 돌게 되는거라, 사이클을 1회만 도는 걸로 변경했습니다.
- chartObj 값 자체가 `undefined` 라면, `Object.entries` 를 읽지 않고 건너뛰게 했습니다.